### PR TITLE
[PRG-2025] Open cfp and sponsor improvements

### PR DESCRIPTION
**Pull request by EtssMM**
Open CFP
Improve sponsor page  - depends on https://github.com/devopsdays/devopsdays-assets/pull/219

### CODE REVIEW

**Review by toshywoshy — APPROVED** [review]

**Comment by toshywoshy** [comment]
LGTM

**Comment by don-code** [comment]
Should the two dates be in 2025? They're in the past (2024), so some of the links on the site intentionally won't show your CFP as open.

File: data/events/2025/prague/main.yml [hunk 1]
```diff
@@ -14,11 +14,11 @@ startdate: 2025-05-22T08:00:00+02:00 # The start date of your event. Leave blank
 enddate: 2025-05-22T18:00:00+02:00 # The end date of your event. Leave blank if you don't have a venue reserved yet.
 
 # Leave CFP dates blank if you don't know yet, or set all three at once.
-cfp_date_start:  # start accepting talk proposals.
-cfp_date_end:  # close your call for proposals.
+cfp_date_start:  2024-01-12T00:00:00+00:00 # start accepting talk proposals.
+cfp_date_end:  2024-01-03T00:00:00+00:00 # close your call for proposals.
 cfp_date_announce:  # inform proposers of status
```
Comment (toshywoshy) [inline data/events/2025/prague/main.yml:19]: As @don-code mentioned, these should be respectivaly
`2024-12-01T00:00:00+00:00`
and
`2025-01-03T00:00:00+00:00`

**Review by toshywoshy — CHANGES_REQUESTED** [review]

File: data/events/2025/prague/main.yml (hunk 1 above)
Comment (EtssMM) [reply data/events/2025/prague/main.yml:19]: stupid me, thanks, should be fixed

**Review by toshywoshy — APPROVED** [review]

**Pull request merged**